pkg/server/plugin/zmq/handler.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017-present Oursky Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zmq
+
+import (
+	"net/http"
+)
+
+type zmqResponseWriter struct {
```

Comment:
Agree, but I am afraid I won't refactor in this PR.

Although I am told the current routing / handler mechanism is "http independent", it does not seems to be true. For example `router.Response` depends on `http.ResponseWriter`, and `router.NewGateway` depends on the concept of "path", which zmq does not have.

Since the current implementation is http orientated, I am going to follow it.